lib/rules.js

```diff
@@ -217,6 +219,40 @@ export const DISTRIBUTE_NEGATIVE_ONE =
 // COLLECT AND COMBINE
 export {default as COLLECT_LIKE_TERMS} from './rules/collect-like-terms'
 
+export const FRACTIONAL_POLYNOMIALS = defineRule(
+    (node) => {
+        let isFractionalPolynomial = false
+        if (query.isMul(node)){
+            const fraction = node.args[1]
+            isFractionalPolynomial = query.isNumber(node.args[0])
+                && query.isDiv(fraction)
+                && isPolynomialTerm(fraction.args[0])
+                && query.isNumber(fraction.args[1])
+        }
+        return isFractionalPolynomial ? {node} : null
+    },
```

Comment:
I was going suggest using a pattern for this but I forgot about https://github.com/semantic-math/math-rules/issues/33.